app/src/layouts/organizationsControl/organizationsControl.jsx

```diff
@@ -33,70 +28,31 @@ import styles from './organizationsControl.scss';
 
 const cx = classNames.bind(styles);
 
-const messages = defineMessages({
-  organization: {
-    id: 'OrganizationsControl.organization',
-    defaultMessage: 'Organization',
-  },
-  all: {
-    id: 'OrganizationsControl.all',
-    defaultMessage: 'All',
-  },
-  allOrganizations: {
-    id: 'OrganizationsControl.allOrganizations',
-    defaultMessage: 'All organizations',
-  },
-});
-
-export const OrganizationsControl = ({ isPopoverOpen, onClick, sidebarType, closeSidebar }) => {
-  const { formatMessage } = useIntl();
-  const organizationName = useSelector(activeOrganizationNameSelector);
+export const OrganizationsControl = ({ isPopoverOpen, onClick, closeSidebar, link, titles }) => {
   const organizationSlug = useSelector(urlOrganizationSlugSelector);
-  const projectName = useSelector(projectNameSelector);
-
-  const titles = {
-    shortTitle: formatMessage(messages.all),
-    topTitle: formatMessage(messages.allOrganizations),
-    bottomTitle: null,
-  };
-
-  const link = { type: ADMINISTRATE_PAGE };
-
-  const getShortTitle = (title) => `${title[0]}${title[title.length - 1]}`;
-
-  switch (sidebarType) {
-    case INSTANCE_LEVEL: {
-      break;
-    }
-    case ORGANIZATION_LEVEL: {
-      titles.shortTitle = getShortTitle(organizationName);
-      titles.bottomTitle = organizationName;
-      break;
-    }
-    default: {
-      titles.shortTitle = getShortTitle(projectName);
-      titles.topTitle = `${formatMessage(messages.organization)}: ${organizationName}`;
-      titles.bottomTitle = projectName;
-      link.type = ORGANIZATION_PROJECTS_PAGE;
-      link.payload = { organizationSlug };
-      break;
-    }
-  }
+  const isInstanceLevel = !organizationSlug;
 
   return (
     <div className={cx('organizations-control-wrapper')}>
-      <button className={cx('organization-block')} onClick={onClick}>
+      <button
+        className={cx('organization-block', { 'no-uppercase': isInstanceLevel })}
```

Comment:
Let's rename CSS class to `short-title`.

Reply:
Done.